modules/machineset-manually-scaling.adoc

```diff
@@ -36,7 +36,27 @@ $ oc edit machineset <machineset> -n openshift-machine-api
 You can scale the MachineSet up or down. It takes several minutes for the new
 machines to be available.
 +
+
+= MachineSet delete policy
+
+MachineSet delete policy defines how priority is assigned to nodes to delete when downscaling a MachineSet. There are three supported options - Random, Newest and Oldest. It default to Random which means that random machines are picked and deleted when downscaling MachineSets. The delete policy can be set according to the use case by modifying the particular MachineSet:
```

Comment:
The MachineSet delete policy

    `Random`, `Newest`, and `Oldest` are the three supported options. The default is `Random`, meaning  that random machines are chosen and deleted when scaling MachineSets down.